src/main/java/com/tabgok/entity/Entity.java

```diff
@@ -0,0 +1,7 @@
+package com.tabgok.entity;
+
+import com.tabgok.harvester.commands.CommandResult;
+
+
```

Comment:
the package name and the class name are really vague, which might be OK but I can't tell without javadoc here explaining what this is :P